.tekton/push.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: pipeline-as-code-on-push
+  annotations:
+    pipelinesascode.tekton.dev/on-event: "[pull_request]"
```

Comment:
Should this be changed to `push` rather than pull request? Otherwise won't that mean `'quay.io/redhat-appstudio/application-service:{{revision}}'` is published before the PR is merged?

Reply:
Yes, sorry about that. I wanted to test the pipeline before merging, but I did that differently. Changed it to push.